downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,218 @@
+
+## Runtime Updates
+
+### Improvements
+
+#### Improved Configurable Variables to Support Enum Types
+
+Values can be provided for configurable variables of `enum` types via command-line arguments and a TOML file.
+
+
+```ballerina
+public enum HttpVersion {
+    HTTP_1_1,
+    HTTP_2
+}
+
+configurable HttpVersion httpVersion = ?;
+```
+
+The value for `httpVersion` can be provided via the `Config.toml` file or as a command-line argument as shown below.
+
+```toml
+[configUnionTypes]
+httpVersion = "HTTP_1_1"
+```
+
+The command-line argument will be as shown below.
+
+```bash
+-ChttpVersion=HTTP_1_1
+```
+
+#### Improved Configurable Variables to Support Map Types
+
+The `configurable` feature is improved to support variables of `map` types. Values can be provided via a TOML file.
+
+For example, if the `map`-typed configurable variables are defined in the following way, 
+
+``` ballerina
+configurable map<string> admin = ?;
+
+type HttpResponse record {|
+    string method;
+    string httpVersion = "HTTP_1_0";
+    map<string> headers;
+|};
+
+configurable HttpResponse response = ?;
+configurable map<string>[] users = ?;
+```
+
+the values can be provided in the `Config.toml` as follows.
+
+```toml
+[admin]
+username = "John Doe"
+mail = "John@hotmail.com"
+location = "LK"
+
+[response]
+method = "POST"
+httpVersion = "HTTP_2_0"
+headers.Status = "200 OK"
+headers.Content-Type = "text/html"
+
+[[users]]
+username = "Jack"
+location = "UK"
+
+[[users]]
+username = "Jane"
+location = "US"
+```
+
+### Breaking Changes
+
+- The `io.ballerina.runtime.api.types.Type#getName` and `io.ballerina.runtime.api.types.Type#getQualifiedName` methods now return an empty string if no name was associated with the type. The `io.ballerina.runtime.api.types.Type#toString` method can be used to get the string representation of a type.
+- `Wait` actions that wait for expressions that are not named workers can return errors now. The eventual type of such wait future expressions is now `T|error` where `T` is the type of the original return value.
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `graphql` Package
+- Added declarative auth configurations
+- Added support for optional types as inputs
+- Added support for returning unions of distinct service objects
+- Added support for inline fragments
+- Added support for enums as input values
+
+### Improvements
+
+#### `cache` Package
+Marked the `cache:Cache` class as an `isolated` class.
+
+#### `file` Package
+Marked the `file:Listener` class as an `isolated` class.
```

Comment:
```suggestion
- Marked the `file:Listener` class as an `isolated` class.
```

Reply:
We don't need bullet points if there is no list and only one point.